core/storage/user/gae_models.py

```diff
@@ -55,25 +57,51 @@ class UserSettingsModel(base_models.BaseModel):
     # migration (to give role to all users) is run.
     role = ndb.StringProperty(
         required=True, indexed=True, default=feconf.ROLE_ID_EXPLORATION_EDITOR)
+    # When the user last agreed to the terms of the site. May be None.
+    last_agreed_to_terms = ndb.DateTimeProperty(default=None)
+    # When the user last logged in. This may be out-of-date by up to
+    # feconf.PROXIMAL_TIMEDELTA_SECS seconds.
+    last_logged_in = ndb.DateTimeProperty(default=None)
+    # Name of a user displayed in android UI. Unlike username, it can be
+    # edited and is unique only among the profiles of the account.
```

Comment:
```suggestion
    # Name of a user displayed in Android UI. Unlike username, it can be
    # edited and is unique only among the profiles of the account.
```

Reply:
Done.